examples/librilight/matching.py

```diff
@@ -82,13 +82,15 @@ def get_params() -> AttributeDict:
 
 
 def _align_worker(
+    index: int,
     sourced_text: SourcedText,
-    transcript: Transcript,
     indexes: Tuple[int, int],
     candidate_matches: List[Tuple[int, int]],
```

Comment:
For the documentation of this: it would be good to have some user-level documentation, e.g. what does the output represent?
Your documentation seems to be aimed at your future self, rather than outsiders :-)

Reply:
Yes, I don't have a stable version yet, so I just write down some notes for myself.